File: tutorials/30_Days_Of_Code/php/d2_Operators.php
<?php

$handle = fopen ("php://stdin","r");
fscanf($handle,"%d",$mealCost);
fscanf($handle,"%d",$tipPercent);
fscanf($handle,"%d",$taxPercent);
$mealCost = round($mealCost);
$tip = $mealCost * $tipPercent / 100;
$tax = $mealCost * $taxPercent / 100;
$tip = round($tip);
$tax = round($tax);
$totalCost = $mealCost + $tip + $tax;
$total = round($totalCost);
print("The total meal cost is ".$total." dollars.");

File: tutorials/30_Days_Of_Code/cpp/d2_Operators.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main(int argc, char **argv)
{
    double mealCost, totalCost, tipPercent, taxPercent;

    cin >> mealCost >> tipPercent >> taxPercent;
    cin.ignore();

    totalCost = mealCost + ((tipPercent/100) * mealCost) + ((taxPercent/100) * mealCost);
    cout << "The total meal cost is " << round(totalCost) << " dollars." << endl;

    return 0;
}